src/test/java/org/folio/oaipmh/helpers/RepositoryConfigurationHelperTest.java

```diff
@@ -45,22 +45,17 @@
   @BeforeAll
   static void setUpOnce(Vertx vertx, VertxTestContext testContext) {
     okapiHeaders.put(OKAPI_TOKEN, "eyJhbGciOiJIUzI1NiJ9");
-    okapiHeaders.put(OKAPI_URL, "http://localhost:" + mockPort);
     OkapiMockServer okapiMockServer = new OkapiMockServer(vertx, mockPort);
     JsonObject conf = new JsonObject()
```

Comment:
are conf and opt still needed here?

Reply:
You are right, it is not required anymore. Thanks

Reply:
Removed